core/src/Kokkos_Macros.hpp

```diff
@@ -58,6 +58,13 @@
 #include <impl/Kokkos_NvidiaGpuArchitectures.hpp>
 #endif
 
+#include <ciso646>
```

Comment:
`<ciso646>` does not exist anymore in C++20 (https://en.cppreference.com/w/cpp/header/ciso646) . Is it ok to unconditionally include it?

Reply:
```suggestion
#include <iso646.h>
```